README.md

```diff
@@ -1,23 +1,72 @@
-Welcome to the MacAdmins Slack Team.
+#Welcome to the MacAdmins Slack Team.
 
+## Introduction
 This community is a complement to the Freenode IRC channels, the various MDM and package management user forums, the Facebook groups, and the mailing lists and seeks to provide a safe, friendly community for Mac administrators, enthusiasts, and developers to interact with one another.
 
-All are welcome to participate within the Slack instance as long as the tenants of this Code of Conduct are adhered to.
+We hope that you find the community to be enriching to your personal and professional lives, and above all else a place of camaraderie where our collective knowledge and differences are celebrated in a positive, welcoming environment.
+
+Thank you for following this code of conduct. We reserve the right to amend or change the code of conduct at any time and encourage you to periodically review these guidelines to ensure a safe environment for all.
+
+-MacAdmins Slack Administrative Team 
+
+---
+##Code Of Conduct
+
+All are welcome to participate within the Slack instance as long as the tenets of this Code of Conduct are adhered to.
+
+## Summary
+
 
 Abide by Wheaton’s Law: don’t be a dick.
```

Comment:
I added the summary to connect with the original CoC and because some people are able to understand that all of the additional language qualifies as "being a dick."